src/nanodbc.cpp

```diff
@@ -1867,8 +1912,15 @@ void statement::statement_impl::bind_strings(
                 bind_len_or_null_[param][i] = SQL_NTS; // null terminated
         }
     }
+    else
+    {
+        for (std::size_t i = 0; i < elements; ++i)
+        {
+            bind_len_or_null_[param][i] = SQL_NTS;
+        }
+    }
 
-    bind_parameter(param, values, elements, data_type, param_type, parameter_size, scale);
+    bind_parameter(param, values, elements, data_type, param_type, length, scale);
```

Comment:
See https://github.com/lexicalunit/nanodbc/issues/116#issuecomment-256687931 for detailed discussion related to this change above.